shopfloor_mobile/static/wms/src/wms_utils.js

```diff
@@ -276,13 +278,19 @@ export class WMSUtils {
         return options;
     }
     move_line_qty_picker_options(line, override = {}) {
-        const opts = {
-            init_value: line.quantity,
-            available_packaging: line.product.packaging,
+        // DEPRECATED - To drop in next versions
+        // Use v-bind="utils.wms.move_line_qty_picker_props(...)" instead
+        alert("wms_utils.move_line_qty_picker_options is deprecated.");
```

Comment:
ALERT? this has to be logged only :thinking:

Reply:
Please fix this :fire:

Reply:
changed to console.log